### ISSUE TITLE
Reported assert failure in FlutterViewUpdateSemantics/updateSemantics

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#31140]
From the following line:

https://github.com/flutter/engine/blob/master/shell/platform/android/platform_view_android_jni.cc#L96

This report does not contain the original exception, but it appears as that either an exception or JNI failure is being triggered in a small percentage of android applications. google3 issue: b/130556204

**Comment by jonahwilliams** [comment]
Duplicate of https://github.com/flutter/flutter/issues/31139
